README.md

```diff
@@ -806,8 +806,9 @@ Other Style Guides
     const h = function() {};
 
     // good
-    const x = function () {};
+    const x = () => {};
```

Comment:
an arrow function doesn't have the spacing issue this section is concerned with; the fact that `function () {}` is bad doesn't mean it's not useful in this example.

I'm fine adding the arrow, but we need to keep the `function () {}`.

Reply:
@ljharb Yea I was thinking about that after I made the PR - added line 811 with a function that has a longer lexical name

Reply:
@ljharb You're probably right, it's too trivial of an inconsistency to not keep what is currently in the doc